SQLQuery_ZooManagement.sql

```diff
@@ -0,0 +1,164 @@
+CREATE Database ZooManagement;
+
+Use ZooManagement;
+
+CREATE TABLE Species
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Species VARCHAR(150),
+SpeciesNumber INT,
+AnimalsNumber INT,
+);
+
+CREATE TABLE Visitors
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Day VARCHAR(150),
+NumberOfVisitors INT,
+);
+
+CREATE TABLE Tickets
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Price INT,
+Validity VARCHAR(150),
+);
+
+CREATE TABLE Caretakers
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Name VARCHAR(150),
+);
+
+CREATE TABLE ZooMentenance
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+AreaOfWork VARCHAR(150),
+);
+
+CREATE TABLE WorkDistribution
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Caretaker INT REFERENCES Caretakers(Id),
+AreaOfWork INT References ZooMentenance(Id) 

```

Comment:
ZooMentenanceId istead of AreaOfWork. The Id part is explained above. Another best practice is that the foreign key holds the name of the referenced entity otherwise it becomes complicated to understand and work with the database structure.